runtime/getting_started/first_project.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Making a Deno project
+oldUrl: /runtime/manual/getting_started/first_steps/
+---
+
+Deno has many [built in tools](TODO:-cli-link) to make your development
+experience as smooth as possible. One of these tools is the
+[project initializer](TODO:init-link), which creates a new Deno project with a
+basic file structure and configuration.
+
+While you are welcome to use JavaScript, Deno has built-in support for
+[TypeScript](https://www.typescriptlang.org/) as well, so we'll be using
+TypeScript in this guide. If you'd prefer to use JavaScript, you can rename the
+files to `.js` and remove the type annotations.
+
+## Initialize a new project
+
+To initialize a new Deno project, run the following command in your terminal:
+
+```bash
+deno init my_project
+```
+
+This will create a new directory called `my_project` with the following
+structure:
+
+```plaintext
+my_project
+├── deno.json
+├── main_test.ts
+└── main.ts
+```
+
+A `deno.json` file is created to [configure your project](TODO:config-link), and
```

Comment:
Nit: this file has some remaining to-do links

Reply:
Yeah, there's gonna be a lot of todos as I go through, cos I'm still moving stuff about. I'll do a big sweep of them once we've got more files moved across to where they'll be and named as we want em.